network_test.go

```diff
@@ -253,7 +254,7 @@ func testNetworkMachineCNI(t *testing.T, useConfFile bool) {
 	}
 	fctesting.RequiresRoot(t)
 
-	cniBinPath := []string{"/opt/cni/bin", testDataBin}
+	cniBinPath := []string{testDataBin, "/opt/cni/bin"}
```

Comment:
Does /opt/cni/bin have x86 binaries in our BuildKite host? Or is it empty?

Reply:
I think, x86 CNI plugin binaries should be present under `/opt/cni/bin`

Reply:
Can you check the host? It is odd if we have x86 CNI plugins on our ARM host.

Reply:
I checked and not able to find any PATH related to  `/opt/cni/bin`. Previously we changed the PATH from `/opt/cni/bin` which is not there and set the PATH to `./testdata/bin` where we installed arm related Binaries.

Reply:
You meant, we don't have the directory? If so, we don't have to have this change to specifically prioritize `testdata/bin` over `/opt/cni/bin`.

Reply:
Can we just keep like this `cniBinPath := []string{testDataBin}`

Reply:
It may work, but let's do that in another PR.

Reply:
the `test` rule of the Makefile has `testdata_objects` as a dependency, which includes the CNI plugins https://github.com/firecracker-microvm/firecracker-go-sdk/blob/49c83706e3a8993212165c18be4b22c458193800/Makefile#L36-L45

Those plugins will be installed to `testDataBin` regardless of platform, so just `cniBinPath := []string{testDataBin}` should be fine for the sake of testing.